react-app/src/components/ResultSidebar.js

```diff
@@ -23,6 +23,15 @@ function ResultSidebar(props) {
       <ResultHeader />
       <div className="sidebar-interior">
         <Accordion defaultActiveKey="0" flush>
+          <Accordion.Item eventKey={-1}>
+            <Accordion.Header
+              onClick={() => {
+                props.setPatientID(-1);
```

Comment:
Per my other comment about the `[showLogs, setShowLogs]` variable, we should be setting patientID to `null` here, and then set the showLogs variable to `true` since that's what we are doing.